### PR TITLE
HOTFIX. Fixes the massive compile issue

### PR DESCRIPTION
**Pull request by svdgoor**

### CODE REVIEW

File: src/main/java/com/volmit/iris/Iris.java [hunk 1]
```diff
@@ -443,40 +443,38 @@ public void imsg(CommandSender s, String msg) {
...
-        if (id != null && !id.isEmpty()) {
-            dimension = id;
-            Iris.info("Generator ID: " + id + " requested by bukkit/plugin. Assuming IrisDimension: " + id);
+        IrisDimension dim;
+        if (id == null || id.isEmpty()){
+            dim = IrisData.loadAnyDimension(IrisSettings.get().getGenerator().getDefaultWorldType());
+        } else {
+            dim = IrisData.loadAnyDimension(id);
         }
+        Iris.info("Generator ID: " + id + " requested by bukkit/plugin");
```
Comment (cyberpwnn) [inline src/main/java/com/volmit/iris/Iris.java]: This should be debug, its useless information for those not in debug or verbose... but we are slowly dropping support for verbose so it should be debug.

File: src/main/java/com/volmit/iris/Iris.java [hunk 2]
```diff
@@ -443,40 +443,38 @@ public void imsg(CommandSender s, String msg) {
...
+            Iris.proj.downloadSearch(new VolmitSender(Bukkit.getConsoleSender()), id, true);
+            dim = IrisData.loadAnyDimension(id);
+
+            if (dim == null) {
+                throw new RuntimeException("Can't find dimension " + id + "!");
             } else {
                 Iris.info("Resolved missing dimension, proceeding with generation.");
             }
         }
+        Iris.info("Assuming IrisDimension: " + dim.getName());
```
Comment (cyberpwnn) [inline src/main/java/com/volmit/iris/Iris.java]: Debug

**Review by cyberpwnn — CHANGES_REQUESTED** [review]
Idk why this was changed, you basically renamed a variable and moved an if statement down but it does the same thing. Either way ill merge it if you change that stuff to debug.

**Comment by svdgoor** [comment]
Because it broke iris entirely lol.
I can't patch this today and you can't compile master without this patch. I'll try compiling tonight but it'll probably be tomorrow morning. If you want to compile today, please merge this and make those changes, otherwise I'll gladly make the changes

**Comment by svdgoor** [comment]
Just going to note these info calls were info calls before I touched the function (non-modified).
See next commit for requested changes.

**Pull request merged**